### PR TITLE
[WFCORE3470] Configure service display name

### PR DESCRIPTION
**Pull request by dleborgne**
Added an option to configure the service display name during service install (WFLY-9619)

### CODE REVIEW

**Comment by dleborgne** [comment]
I think the patch is ready for review

**Comment by ctomc** [comment]
this is ok to test

File: core-feature-pack/src/main/resources/content/docs/contrib/scripts/service/service.bat [hunk 1]
```diff
@@ -147,10 +148,11 @@ echo Usage:
 echo(
 echo   service install ^<options^>  , where the options are:
 echo(
-echo     /startup                  : Set the service to auto start
+echo     /startup                    : Set the service to auto start
```
Comment (dmlloyd) [inline core-feature-pack/src/main/resources/content/docs/contrib/scripts/service/service.bat]: The spacing was changed here

File: core-feature-pack/src/main/resources/content/docs/contrib/scripts/service/service.bat [hunk 2]
```diff
@@ -195,13 +204,14 @@ echo(
...
 echo                                 default: %LOGLEVEL%
+echo(
 echo     /logpath ^<path^>           : Path of the log
 echo                                 default depends on domain or standalone mode
 echo                                 /base applies when /logpath is not set.
 echo                                   %JBOSS_HOME%\domain\log
 echo                                   %JBOSS_HOME%\standalone\log
 echo(
-echo     /debug                    : run the service install in debug mode
+echo     /debug                      : run the service install in debug mode
```
Comment (dmlloyd) [inline core-feature-pack/src/main/resources/content/docs/contrib/scripts/service/service.bat]: Spacing also changed here

**Comment by dmlloyd** [comment]
We require that all contributions be made under the terms of the MIT License, http://www.opensource.org/licenses/mit-license.php. Do you agree to the these terms?

**Comment by dleborgne** [comment]
Yes, I agree that my contributions are made under the terms of the MIT License.

**Comment by bstansberry** [comment]
I moved the JIRA for this to WFCORE as the Fix Version tracking will be against a WildFly Core release. I'm going to change the title of this PR to use the new JIRA # WFCORE-3470 so the git log will include it in the history (to help find this PR), but there's no need to change the commit message. If any goes to WFLY-9619 JIRA will redirect them to WFCORE-3470.

**Pull request merged**